src/main/kotlin/com/danix/anaconda/QuoteController.kt

```diff
@@ -14,6 +14,7 @@ import javax.servlet.http.HttpServletRequest
 import javax.ws.rs.core.Response.Status
 
 @RestController
+@RequestMapping("/api")
```

Comment:
maybe this could be added as server.contextPath=/api configuration so in ResourceServerConfiguration it won't be attached to any specific controller.